apps/vetrina-test/spago.dhall

```diff
@@ -0,0 +1,29 @@
+{-
```

Comment:
Same as above

Reply:
The name of the project should be at least fixed 😅